### PR TITLE
E2e tests to verify user details on logging in

### PR DESCRIPTION
**Pull request by shakirandagire**
@AmTryingMyBest Could you help me review this PR that handles issue [717](https://github.com/ushahidi/platform-client-mzima/issues/717)

### CODE REVIEW

File: e2e-testing/cypress/functions/LoginFunctions.js [hunk 1]
```diff
@@ -25,6 +25,15 @@ class LoginFunctions {
     cy.get(LoginLocators.loginButton).click();
   }
 
+  check_user_details_correct(){
```
Comment (theobadha) [inline e2e-testing/cypress/functions/LoginFunctions.js]: I don't see where this method is used. @shakirandagire 
Was this implementation complete?

Comment (shakirandagire) [reply e2e-testing/cypress/functions/LoginFunctions.js]: I had completed the implementation and put the method in the cypress file. Let me check and see what happenned

**Comment by shakirandagire** [comment]
@AmTryingMyBest The tests are passing on my local but failing on cypress cloud. This is because the admin username is not being found. I am not sure how we can add it to the tests running online.

**Comment by theobadha** [comment]
@shakirandagire okay, let me check it out.

**Comment by theobadha** [comment]
@shakirandagire we needed to add "ush_admin_name" as a secret into github with the appropriate value. Also changed viewport for the fields getting verified to be visible. 
Pushed the changes. Should pass now.

**Pull request merged**